taskcluster-hook-pipeline-start.json

```diff
@@ -0,0 +1,62 @@
+{
+    "schedule": [
+        "* */5 * * * *"
+    ],
+    "metadata": {
+        "description": "",
+        "name": "BugBug data pipeline",
+        "owner": "mcastelluccio@mozilla.com"
+    },
+    "task": {
+        "created": {
+            "$fromNow": "0 seconds"
+        },
+        "deadline": {
+            "$fromNow": "2 hours"
+        },
+        "expires": {
+            "$fromNow": "1 month"
+        },
+        "extra": {},
+        "metadata": {
+            "description": "",
+            "name": "BugBug data pipeline",
+            "owner": "mcastelluccio@mozilla.com",
+            "source": "https://github.com/mozilla/bugbug"
+        },
+        "payload": {
+            "artifacts": {},
+            "cache": {},
+            "capabilities": {},
+            "command": [
+                "/hello"
+            ],
+            "env": {},
+            "features": {
+                "taskclusterProxy": true
+            },
+            "image": "hello-world",
+            "maxRunTime": 7200
+        },
+        "priority": "normal",
+        "provisionerId": "aws-provisioner-v1",
+        "retries": 5,
+        "routes": [
+            "index.project.relman.services.project.bugbug.data_pipeline.latest"
+        ],
+        "schedulerId": "-",
+        "scopes": [
```

Comment:
Instead of defining all scopes here, we can use `assume:hook-id:project-relman/bugbug` and define all the required scopes in the hook-id:project-relman/bugbug role.